source/__tests__/integration/new-profile.test.js

```diff
@@ -41,19 +41,32 @@ describe('Adding and removing a new Profile', () => {
         await expect(popover).toBe('Link to more information about profile')
       })
 
-      it('validates with checkURL', async () => {
+      it('with function checkURL', async () => {
```

Comment:
i don't understand this description change.  The old description described the expected behavior;  the new description is describing code specifics.   Can you help me understand your intent?